transportable-udfs-trino-plugin/src/test/java/com/linkedin/transport/trino/TransportPluginTest.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright 2023 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.transport.trino;
+
+import io.trino.server.testing.TestingTrinoServer;
+import io.trino.spi.Plugin;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import static com.google.common.collect.Iterables.getOnlyElement;
+
+
+public class TransportPluginTest {
+
+  @Test
+  public void testTransportPluginInitialization() {
+    TestingTrinoServer server = TestingTrinoServer.create();
+    Plugin plugin = new TransportPlugin();
+    server.installPlugin(plugin);
+    server.createCatalog("LINKEDIN", "TRANSPORT");
+    Assert.assertTrue(getOnlyElement(plugin.getConnectorFactories()) instanceof TransportConnectorFactory);
+  }
+}
```

Comment:
can we assert that the UDFs are actually able to be loaded?

it would be nice also to have a test which can assert that each UDF is in a separate classloader -- maybe load two UDFs and grab the classloader for each and assert that they are different?

Reply:
+1 @yiqiangin

```
LocalQueryRunner _runner = ...;
_runner.installPlugin(new TransportPlugin());
_runner.createCatalog("LINKEDIN", "TRANSPORT", ImmutableMap.of(<some config>...));
```

Then we can run some query to verify the functions exist.

Reply:
Yes, I change this test to initialize `TransportPlugin` with a LocalQueryRunner. So the success of loading of UDFs are verified by successfully executing a query with UDF via LocalQueryRunner as shown in https://github.com/linkedin/transport/pull/133

For the test which can assert that each UDF is in a separate classloader, I am still trying to find out a good way to implement this in Transport side.